apps/teams-test-app/src/components/MediaAPIs.tsx

```diff
@@ -1,25 +1,82 @@
-import { media } from '@microsoft/teams-js';
+import { media, SdkError } from '@microsoft/teams-js';
 import React, { ReactElement } from 'react';
 
 import { noHostSdkMsg } from '../App';
 import { ApiWithoutInput, ApiWithTextInput } from './utils';
 
+const mediaHelper = (item: string): string => {
+  let output = '';
+  let len = 20;
+  if (item) {
+    len = Math.min(len, item.length);
+    output = item.substr(0, len);
+  }
+  return output;
+};
+
+const captureImageHelper = (file: media.File): string => {
+  const content = mediaHelper(file.content);
+  const output =
+    'format: ' + file.format + ', size: ' + file.size + ', mimeType: ' + file.mimeType + ', content: ' + content;
+
+  return output;
+};
+
+const selectMediaHelper = (medias: media.Media[]): string => {
+  let message = '';
+  for (let i = 0; i < medias.length; i++) {
+    const media: media.Media = medias[i];
+    const preview = mediaHelper(media.preview);
+    message +=
+      '[format: ' +
+      media.format +
+      ', size: ' +
+      media.size +
+      ', mimeType: ' +
+      media.mimeType +
+      ', content: ' +
+      media.content +
+      ', preview: ' +
+      preview +
+      '],';
+  }
+  return message;
+};
+
+const getMediaHelper = (blob: Blob, setResult: (result: string) => void): void => {
+  const reader = new FileReader();
+  reader.readAsDataURL(blob);
+  reader.onloadend = () => {
+    if (reader.result) {
+      // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+      setResult('Received Blob (length: ' + (reader.result as any).length + ')');
+    }
+  };
+};
+
```

Comment:
Thoughts on moving the helper functions into a separate file `mediaHelpers.ts` under the `utils` folder